### ISSUE TITLE
Documentation with all hooks and callback methods

### ISSUE THREAD

**Issue by jonasdumas** [kettanaito/react-advanced-form#125]
Is it possible to have a list of all available hooks and callback methods for Forms and Fields?

**Comment by kettanaito** [comment]
Hi.

All this info should be currently available in the docs. Maybe the structure of the latter is not very efficient to find those hooks and callbacks.

Would it be easier if I added "Hooks" section alongside existing "Methods", and list all hooks there?

**Comment by kettanaito** [comment]
@jonasdumas I've added separate "Methods" and "Callbacks" sections to the [Documentation](https://kettanaito.gitbooks.io/react-advanced-form/). Indeed, there were a few things missing. Now it should have all the available callbacks and methods, hopefully.

Note that `Field` instances accept any event handlers you would pass to form elements usually (`onInput`, `onKeyDown`, etc.). I will find a way to document this in the future.

Closing.
